argus/backend/service/testrun.py

```diff
@@ -91,6 +95,8 @@ def get_runs_by_id(self, test_id: UUID, runs: list[UUID]):  # FIXME: Not needed,
         return response
 
     def change_run_status(self, test_id: UUID, run_id: UUID, new_status: TestStatus):
+        if not test_id:
```

Comment:
> this checker seems useless without checking ArgusTest.get(id=test_id) output.
> 
> need to catch empty output or/and possible exception at test = ArgusTest.get(id=test_id) as well, if test_id does not exist and raise TestRunServiceException if needed

The exception there would be very early and it will just say `"None" is not an allowed filter value`, however we should indeed handle `ArgusTest.DoesNotExist` exception. Fixed.

I've added handler for the DoesNotExist exception, but I'm keeping the check as is, as CQLEngine will just raise `CQLEngineException` which is a parent of `DoesNotExist` exception if you give a NULL filter value.